Jenkinsfile

```diff
@@ -32,13 +32,18 @@ pipeline {
       steps {
         timeout(time: 10, unit: 'MINUTES') {
```

Comment:
@michaelneale If you know of a way to wrap this `timeout` inside of the `when` blocks, I'd love to know how to do that. I was able to accomplish what I set out to, but it felt a little dirty doing it this way.

Reply:
Hrm - maybe @abayer knows. I know a recent edition of Declarative allowed input statements to be put in the `when` section, but not sure if timeout can be wrapped in there?

Reply:
You can put the `timeout` in the `stage`’s `options` block...is that what you’re looking for?

Reply:
Yes, it is - https://jenkins.io/doc/book/pipeline/syntax/#stage-options combined with https://jenkins.io/doc/book/pipeline/syntax/#input should be exactly what you want.

Reply:
thanks @abayer !

Reply:
Oh cool. Nice tip. Thanks for your help guys! Is that available on Gauntlet? I sounded like the Gauntlet masters may be on an old version of Jenkins or something like that?

I ran into some issues before trying to follow the documentation and it wasn't working on Gauntlet.

Reply:
yeah I think it is pretty new, and that tends to use the stable releases, so perhaps use this as is now. At least in future it would be simpler!